yafti/__init__.py

```diff
@@ -13,3 +13,5 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+__version__: str = "0.9.0-beta1"
+__name__: str = "yafti"
```

Comment:
This shouldn't be needed. It's already tracked in the pyproject.toml

Reply:
just personal preference I guess. will remove if you hate it.